src/events/crawler/scrapers/IND/index.js

```diff
@@ -0,0 +1,43 @@
+import * as fetch from '../../lib/fetch.js';
+import * as parse from '../../lib/parse.js';
+import * as rules from '../../lib/rules.js';
+import populationState from './populationState.json';
+
+const scraper = {
+  country: 'IND',
+  url: 'https://www.mohfw.gov.in/',
+  type: 'table',
+  aggregate: 'state',
+
+  async scraper() {
+    this.url = 'https://www.mohfw.gov.in/';
+    const $ = await fetch.page(this.url);
+
+    const $table = $('table[style="font-weight:bold"]');
+    $('table tr')
+      .slice(-2)
```

Comment:
This could get brittle if they decide to remove that row, instead, why don't you check for the `colspan` property on the first `td` inside of the table?